installing/installing_oci/installing-oci-agent-based-installer.adoc

```diff
@@ -1,6 +1,6 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="installing-oci-agent-based-installer"]
-= Installing a cluster OCI by using the Agent-based Installer 
+= Using the Agent-based Installer to install a cluster on OCI
```

Comment:
I'd suggest spelling out OCI as Oracle Cloud Infrastructure (so that it's not confused with Open Container Initiative).

Also Agent-based Installer is used primarily for restricted network (i.e. disconnected/air-gapped) deployments.

Reply:
I changed the title to "The Agent-based Installer for cluster installation on Oracle Cloud Infrastructure" to reduce it's length on a portal table of contents.

Reply:
ditto